docs/resampling.ipynb

```diff
@@ -110,7 +110,7 @@
     "catalog_hsc, bg_rms_hsc = makeCatalog(data_hsc)\n",
```

Comment:
In `makeCatalog` and elsewhere: why are all those `byteswap().newbyteorder()` necessary?

Reply:
See https://sep.readthedocs.io/en/v1.0.x/tutorial.html#Finally-a-brief-word-on-byte-order. This is an issue with the way that astropy handles endianness: http://docs.astropy.org/en/stable/known_issues.html#bug-with-unicode-endianness-in-io-fits-for-big-endian-processors

Reply:
idk
Something used to complain about the type, but now it seems fine.